Cargo.toml

```diff
@@ -22,13 +30,13 @@ sendgrid = ">=0.7.0"
 serde = ">=1.0"
 serde_derive = ">=1.0"
 serde_json = ">=1.0"
+slog = "2.2.3"
+slog-async = "2.3.0"
+slog-mozlog-json = "0.1.0"
+slog-term = "2.4.0"
 validator = ">=0.6.3"
 validator_derive = ">=0.6.5"
 
-[[bin]]
-name = "service"
-path = "src/service_main.rs"
-
-[[bin]]
-name = "queues"
-path = "src/queues_main.rs"
+[dependencies.chrono]
```

Comment:
custom dependency also changed.  *shrugs*

Reply:
Fwiw I *kind* of prefer the dependencies all in one big list like they were before. Also it keeps this diff focused on the substantive change, which is supporting mozlog.